src/index.html

```diff
@@ -6,12 +6,155 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet"
+      href="./style.css"
+    >
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get"
+    >
+      <fieldset class="fields">
+        <legend>Personal information</legend>
+        <div class="item">
+          <label for="surname">Surname:</label>
+          <input
+            id="surname"
+            type="text"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="item">
+          <label for="name">Name:</label>
+          <input id="name"
+            type="text"
+            autocomplete="off"
+            minlength="2"
+          >
```

Comment:
if you put attributes on new line, make sure to put EVERY attribute on a new line, including the first one
```suggestion
          <input
            id="name"
            type="text"
            autocomplete="off"
            minlength="2"
          >
```

Reply:
solved. thank you for your advice. now you can check and see it's as you wanted